lib/lndclient/LndClient.ts

```diff
@@ -290,6 +289,21 @@ class LndClient extends BaseClient {
     return this.unaryCall<lndrpc.SendToRouteRequest, lndrpc.SendResponse>('sendToRouteSync', request);
   }
 
+  /**
+   * Subscribe to invoices.
+   */
+  private subscribeInvoices = (): void => {
+    if (this.invoiceSubscription) {
+      this.invoiceSubscription.cancel();
+    }
+
+    this.invoiceSubscription = this.lightning.subscribeInvoices(new lndrpc.InvoiceSubscription(), this.meta)
+    .on('error', (error) => {
+      this.setStatus(ClientStatus.Disconnected);
```

Comment:
I think it would make sense to set `this.invoiceSubscription` to `undefined` here, so we don't try to close a stream that has already been canceled.